pkg/kor/kor_test.go

```diff
@@ -130,3 +130,55 @@ func TestGetKubeClientFromInput(t *testing.T) {
 		t.Errorf("Expected valid clientSet")
 	}
 }
+
```

Comment:
Why not create resource specific tests?

Reply:
What do you mean by resource specific?

Reply:
For example create a configmap with the name /regex from the exceptions json

Reply:
if I'm understanding correctly, you're suggesting to test with the objects themselves and not just the name,

If that's truly the case, I'm not sure I see a point in doing so. The functionality of resource exceptions should be the same over all objects and thus only it should be tested and not the way the parameters to said procedure are provided.

Apologies for my delayed response,

Reply:
OK.
Makes sense.
I'll make some order in this kor file soon and split it off, but logically this does cover your code